lua/astrocommunity/pack/godot/init.lua

```diff
@@ -1,3 +1,40 @@
+local function is_godot_project(path)
+  -- Start with the current directory
+  local current_path = path
+
+  -- Keep traversing up until we reach the root directory
+  while current_path and current_path ~= "" do
+    -- Check if project.godot exists in the current directory
+    if vim.uv.fs_stat(current_path .. "/project.godot") then return true end
+
+    -- Move up one directory level
+    local parent_path = vim.fn.fnamemodify(current_path, ":h")
+
+    -- If we've reached the root or haven't changed directories, stop
+    if parent_path == current_path then break end
+
+    current_path = parent_path
+  end
+
+  return false
+end
```

Comment:
Perhaps add in a check for a parent git folder, and stop when thats reached?

Reply:
Yeah, will update the implementation to rely on the same logic as godot-server.nvim